src/Features/LanguageServer/Protocol/DefaultCapabilitiesProvider.cs

```diff
@@ -74,7 +73,7 @@ public ServerCapabilities GetCapabilities(ClientCapabilities clientCapabilities)
 
             capabilities.SemanticTokensOptions = new SemanticTokensOptions
             {
-                Full = new SemanticTokensFullOptions { Delta = true },
+                Full = false,
```

Comment:
I think it might be useful to make a note in the code for future unwary travelers of why roslyn doesn't support full handling